src/api/svg.ts

```diff
@@ -0,0 +1,537 @@
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+      scale: element.svgAttributes.scale,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async image(element) {
+    page.drawImage(await page.doc.embedPng(element.svgAttributes.src), {
+      x: options.x + element.svgAttributes.x,
+      y: options.y - element.svgAttributes.y,
+      width: element.svgAttributes.width,
+      height: element.svgAttributes.height,
+      opacity: element.svgAttributes.fillOpacity,
+      xSkew: element.svgAttributes.skewX,
+      ySkew: element.svgAttributes.skewY,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async rect(element) {
+    if (!element.svgAttributes.fill && !element.svgAttributes.stroke) return;
+    page.drawRectangle({
+      x: options.x + element.svgAttributes.x,
+      y: options.y - element.svgAttributes.y,
+      width: element.svgAttributes.width,
+      height: element.svgAttributes.height,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+      xSkew: element.svgAttributes.skewX,
+      ySkew: element.svgAttributes.skewY,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async ellipse(element) {
+    page.drawEllipse({
+      x: options.x + element.svgAttributes.x + element.svgAttributes.cx,
+      y: options.y - element.svgAttributes.y - element.svgAttributes.cy,
+      xScale: element.svgAttributes.rx,
+      yScale: element.svgAttributes.ry,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async circle(element) {
+    page.drawCircle({
+      x: options.x + element.svgAttributes.x + element.svgAttributes.cx,
+      y: options.y - element.svgAttributes.y - element.svgAttributes.cy,
+      size: element.svgAttributes.r,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+    });
+  },
+});
+
+const transform = (
+  { x, y }: Position,
+  name: string,
+  args: number[],
+): Position => {
+  let angle;
+  let tempResult;
+  switch (name) {
+    case 'scale':
```

Comment:
Here, the converter should be changed so that:
```javascript
newConverter = {
  point: (x, y) => {const pt = converter.point(x, y); return {
    x: pt.x * xScale,
    y: pt.y * yScale
  }},
  size: (w, h) => {const size = converter.size(w, h); return {
    width: size.x * xScale,
    height: size.y * yScale
  }}
}
```